### ISSUE TITLE
Nombre de vues

### ISSUE THREAD

**Issue by B4rb3rouss** [BlogoText/blogotext#349]
Dans les graphiques de l'accueil du panel d'administration, j'ai pensé qu'il serait pertinent d'avoir le nombre de "vues" par mois.
Plusieurs questions se posent cependant : 

- Cela risque-t-il de "casser" une installation existante si on ajoute un compteur pour chaque article?
- Niveau performances, incrémenter dans la base de données à chaque vue, c'est moyen non? 

Bref, quelques réflexions le temps que les addons partie admin soient dispos. :)

**Comment by remrem** [comment]
Je ne suis pas trop chaud pour ce genre de fonctionnalité directement dans le core, si ça sort en addon, pas de problème pour moi.

**Comment by B4rb3rouss** [comment]
Oui, ça sera en addon bien sûr, les changements que ça apporte niveau perfs ne doivent pas être intégrés par défaut.
Il faut d'ailleurs penser à un addon pour installer des addons, mais c'est un autre problème.
